### PR TITLE
Added my theme XCore

### PR DESCRIPTION
**Pull request by bumblebritches57**

### CODE REVIEW

**Comment by bumblebritches57** [comment]
I was wondering what the correct key was! I'll fix that asap.

File: packages.json [hunk 1]
```diff
@@ -2311,7 +2311,12 @@
         "url": "https://raw.githubusercontent.com/sailorhg/fairyfloss/gh-pages/Fairyfloss.dvtcolortheme",
         "description": "A pastel/candy/daydream fairyfloss theme by @sailorhg.",
         "screenshot": "https://cloud.githubusercontent.com/assets/498212/15276890/9db3f44a-1ac2-11e6-9608-de79505c55ce.png"
-      }
+        }, {
```
Comment (guillaumealgis) [inline packages.json]: For consistency, please move the opening brace to a new line.

**Comment by guillaumealgis** [comment]
Thanks for fixing @bumblebritches57, I would be nice if you could move the brace on a new line, and then we're good to go 👍 

(didn't mean to approve the changes yet. this new UI is somewhat confusing)

**Comment by bumblebritches57** [comment]
Alright. I'm pushing that fix rn.

**Comment by bumblebritches57** [comment]
It's not showing up here, the other day I did re-clone it, so you'll either need to go to https://github.com/bumblebritches57/alcatraz-packages

Or, how do I update this PR?

**Comment by guillaumealgis** [comment]
@bumblebritches57 usually you just have to push more commits on the branch you created for the PR. No need to re-clone anything.

If you want to keep the commit history clean, you can also rewrite it (with `git commit --amend` or `git rebase`) and `git push -f` on your branch.

See https://stackoverflow.com/questions/9790448/how-to-update-a-pull-request

Don't worry about it though, the brace is no big deal, and the JSON needs a cleanup anyway. I'll do it 👍

**Comment by guillaumealgis** [comment]
Thank you @bumblebritches57 !

**Pull request merged**